### PR TITLE
Customize action button

### PR DESCRIPTION
**Pull request by ClementGautier**
| Q                 | A
| ----------------- | ---
| Specs             | N/A
| Behats            | N/A
| Blue CI           | running
| Changelog updated | N/A
| Review and 2 GTM  | todo
| Micro Demo (PO)   | todo
| Migration script  | N/A
| Tech Doc          | N/A

# Goal
The goal here was to customize our buttons to be able to make this kind of buttons in the datagrids:

New buttons

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 1]
```diff
@@ -79,6 +100,21 @@
   &:hover {
     background-color: darken(@color-new-green, 10%);
   }
+
+  &.tiny {
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: duplicated code. You could do a class for that

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 2]
```diff
@@ -192,6 +228,21 @@
   &:hover {
     background-color: darken(@color-new-red, 10%);
   }
+
+  &.tiny {
```
Comment (juliensnz) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: same here

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less (hunk 1 above)
Comment (ClementGautier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: ask Stephane about that.

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less (hunk 2 above)
Comment (ClementGautier) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: same

**Comment by juliensnz** [comment]
GTM

**Comment by jjanvier** [comment]
gtm

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 3]
```diff
@@ -5,6 +5,22 @@
   margin-left: 4px;
 }
 
+.grid .action-cell a.action.btn-tiny,
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: why adding `a` here ?

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 4]
```diff
@@ -430,10 +452,40 @@
...
   padding: 5px 10px;
-  margin: 5px;
   border-radius: 2px;
   background-color: @akeneo-light;
   border: solid 1px @akeneo-grey;
   color: @slate-grey;
   white-space: nowrap;
 }
+
+.grid .action-cell a.action.btn {
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: same

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 5]
```diff
@@ -430,10 +452,40 @@
...
+  opacity: 0;
+
+  i {
+    text-align: center;
+    border: none;
+    border-radius: 2px 0 0 2px;
+  }
+}
+
+tr:hover>.action-cell a.action.btn {
```
Comment (phaseinducer) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: same

**Comment by phaseinducer** [comment]
GTM after fix

**Pull request merged**